addons/website/static/src/snippets/s_facebook_page/000.js

```diff
@@ -1,20 +1,26 @@
 odoo.define('website.s_facebook_page', function (require) {
 'use strict';
 
-var publicWidget = require('web.public.widget');
-var utils = require('web.utils');
+const publicWidget = require('web.public.widget');
+const utils = require('web.utils');
+const { debounce } = require("@web/core/utils/timing");
 
 const FacebookPageWidget = publicWidget.Widget.extend({
     selector: '.o_facebook_page',
     disabledInEditableMode: false,
 
+    /**
+     * @constructor
+     */
+    init() {
+        this._super(...arguments);
+        this.previousWidth = 0;
```

Comment:
I prefer you add this in the `start();` to avoid too many changes in stable version.